www/class/centreon-partition/partEngine.class.php

```diff
@@ -424,31 +417,9 @@ public function migrate($table, $db)
     public function updateParts($table, $db)
     {
         $tableName = $table->getSchema() . "." . $table->getName();
-        if (!$table->exists()) {
```

Comment:
Why delete this verification ?

Reply:
Because we test if table exists with ```isPartitioned``` method.